README.md

```diff
@@ -65,25 +65,29 @@ sudo yum install gcc-c++ automake libtool pkgconfig boost-devel \
 To install on a FreeBSD system, use
 
 ```sh
-pkg install devel/git devel/autoconf devel/automake devel/gmake devel/libtool \
+pkg install devel/git devel/cmake
   textproc/expat2 graphics/geos graphics/proj databases/postgresql94-client \
   devel/boost-libs lang/lua52 devel/pkgconf
 ```
 
-Then you should be able to bootstrap the build system:
+CMake 2.8.7 is needed to build oslm2pgsql. First generate the makefiles in separate folder:
 
-    ./autogen.sh
+    mkdir build && cd build
+    cmake .. -G "Unix Makefiles"
```

Comment:
On some systems it does... But maybe it was on Windows :)

Will try to remove it from travis.